woocommerce/woocommerce
# Remove adding and managing products note

### All Submissions:






### Changes proposed in this Pull Request:

Remove the note about adding and managing products.

Part of #34930  .






### How to test the changes in this Pull Request:



1. Create at least 1 published product on a store that is at least 3 days old (or manually adjust the `woocommerce_admin_install_timestamp` option to this timeframe)
2. Install a plugin like [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)
3. Under Tools -> Cron Events, run `wc_admin_daily`
4. Make sure the note with a title of "Adding and Managing Products" has not been added



### Other information:




### FOR PR REVIEWER ONLY:
